.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.tree.indent": 20
+}
```

Comment:
vscode settings file should not be part of the PR

Reply:
done